sympy/utilities/iterables.py

```diff
@@ -19,6 +19,40 @@
     multiset_partitions_taocp, list_visitor, MultisetPartitionTraverser)
 
 
+def is_palindromic(s, i=0, j=None):
+    """return True if the sequence is the same from left to right as it
+    is from right to left in the whole sequence (default) or in the
+    Python slice ``s[i: j]``; else False.
+
+    EXAMPLES
+    ========
```

Comment:
This should be capitalised but not all-caps.

Reply:
Ughh...how did this creep into my editing habits!